packages/patterns-react/src/patterns/index.js

```diff
@@ -8,3 +8,4 @@
 export * from './LeadSpace';
 export * from './ListSection';
 export * from './SimpleLongForm';
+export * from './PictogramArray';
```

Comment:
minor nitpick, can this be in alphabetical order with the other exports?

Reply:
I've ordered it properly in the last commit.